docs/guides/reference/dataset_maturity_guide.md

```diff
@@ -25,7 +25,7 @@ DEA produces ARD data to three maturity levels:
 {#nrt}
 ### Near Real-Time (NRT)
 
-**Near Real-Time (NRT)** is a rapid ARD product produced within 48 hours (and often less than 24 hours) of image capture. NRT 
+**Near Real-Time (NRT)** is a rapid ARD product produced within 48 hours (and often closer to 24 hours) of image capture. NRT 
```

Comment:
Here's a wording suggestion - it's a bit longer but I think it is slightly easier to read.

```md
**Near Real-Time (NRT)** is a rapid ARD product that is produced within 48 hours of image capture. In most cases, it is produced in just 24 hours. NRT 
```